src/webviews/webview-side/ipywidgets/kernel/index.ts

```diff
@@ -55,7 +55,7 @@ class WidgetManagerComponent {
             cdnsUsed: this.widgetsCanLoadFromCDN,
             isOnline: data.isOnline,
             timedout: data.timedout,
-            error: data.error
+            error: JSON.stringify(data.error)
```

Comment:
Changes to ensure all of the error information is sent, if we don't do this at the extension host end we get `[Object object]` & don't get any of the error information in the logs.